apps/src/panels/PanelsView.tsx

```diff
@@ -283,9 +286,13 @@ const PanelsView: React.FunctionComponent<PanelsProps> = ({
                   className={classNames(
                     'icon',
                     styles.bubble,
-                    index === currentPanelIndex
-                      ? styles.bubbleCurrent
-                      : styles.bubbleNotCurrent
+                    background === 'light'
+                      ? index === currentPanelIndex
+                        ? styles.bubbleCurrentLight
+                        : styles.bubbleNotCurrentLight
+                      : index === currentPanelIndex
+                      ? styles.bubbleCurrentDark
+                      : styles.bubbleNotCurrentDark
```

Comment:
Similar comment here (interpolate background directly into style name to simplify logic)

Reply:
Good call.